design-documents/graph-ql/coverage/CartAddressOperations.graphqls

```diff
@@ -0,0 +1,84 @@
+type Query {
+    addBillingAddressToCart(input: AddBillingAddressToCartInput): AddBillingAddressToCartOutput
+    addShippingAddressToCart(input: AddShippingAddressToCartInput): AddShippingAddressToCartOutput
+}
+
+input AddBillingAddressToCartInput {
```

Comment:
Suggestion: rename to setBillingAddressOnCart (to show that there can only be one)